### PR TITLE
AG-978 zero scores showing as data unavailable

### PR DESCRIPTION
**Pull request by sagely1**

### CODE REVIEW

File: src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.ts [hunk 1]
```diff
@@ -105,13 +105,13 @@ export class GeneSoeChartsComponent implements OnInit {
       this._gene?.overall_scores || ({} as OverallScores);
 
     if ('Genetic Risk Score' === name) {
-      return scores['genetics_score'] || 0;
```
Comment (sagely1) [inline src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.ts:108]: This is the underlying issue it seems.  Null scores have meaning to the charts.

File: src/app/features/charts/components/score-chart/score-chart.component.spec.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ describe('Component: Chart - Score', () => {
   });
 
   it('should display message if not data', () => {
-    expect(component.score).toEqual(0);
+    expect(component.score).toEqual(null);
     expect(element.querySelector('.chart-no-data')).toBeTruthy();
   });
```
Comment (nickgros) [inline src/app/features/charts/components/score-chart/score-chart.component.spec.ts]: You could consider also verifying that a 0 score is treated as an actual value, if you want to capture both sides of the edge case. Might be kind of awkward with how this test is set up.

File: src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.scss [hunk 3]
```diff
@@ -52,12 +52,12 @@
...
-    padding-bottom: 15px;
-  }
+  // .score-chart {
+  //   min-width: 320px;
+  // }
+
+  // .chart-body {
+  //   padding-top: 15px;
+  //   padding-bottom: 15px;
+  // }
```
Comment (nickgros) [inline src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.scss]: Delete if this won't be used in the near-future 🙂

**Review by nickgros — APPROVED** [review]
Few suggestions but otherwise LGTM!

File: src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.scss (hunk 3 above)
Comment (sagely1) [reply src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.scss]: Might as well delete this since I think this whole chart will be rewritten.

File: src/app/features/charts/components/score-chart/score-chart.component.spec.ts (hunk 2 above)
Comment (sagely1) [reply src/app/features/charts/components/score-chart/score-chart.component.spec.ts]: Great suggestion.  I added zero and valid value cases.
